doc/nrf/security/tfm.rst

```diff
@@ -87,13 +87,17 @@ Logging
 *******
 
 TF-M employs two UART interfaces for logging: one for the secure part (MCUboot and TF-M), and one for the non-secure application.
-The logs arrive on different COM ports on the host PC.
+By default, the logs arrive on different COM ports on the host PC. See the :ref:`ug_tfm_manual_VCOM_connection` for more details.
 
+Alternatively, you can configure the TF-M to connect to the same UART as the application by using the :konfig:option:`CONFIG_TFM_SECURE_UART_SHARE_INSTANCE` Kconfig option.
+Setting this Kconfig option does not require manual connection to the Virtual COM ports, making the TF-M logs visible on the application VCOM.
```

Comment:
Something about this sentence still seems off, but I can't quite wrap my head around it.

Reply:
That flows a lot better, yeah.